src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -380,9 +388,87 @@ You can also query for all existing streams. Here's an example:
+    1. Navigate [here](https://github.com/newrelic/metadata-setting-lambda-function), clone the repository locally, and follow the provided steps in the README file to generate a ZIP file containing the lambda function.
+    2. Create an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) for the function.
+    * When [creating the role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html#roles-creatingrole-service-console), be sure to set the trusted entity type as "AWS Service", with "Lambda" as your use case.
+    * This role must have a policy with at least these permissions: `s3:PutObject` and `s3:GetObject`.
+    3. Navigate to the [Lambda functions page](https://console.aws.amazon.com/lambda/home#/functions) in AWS
+    4. Click the **Create function** button.
+    5. Select the Java 11 runtime environment.
+    6. Click **Change default execution role**, then select **Use an existing role**. Enter the role you created in step 2 here.
+    7. Scroll down and click the **Create function** button.
+    8. Once the function has been created, click **Upload from**, and select **.zip or .jar file** from the dropdown.
+    9. Click **Upload** from the box that pops up, and select the ZIP file created in step 1.
+    10. Once the upload has finished, click **Save** to exit the pop-up box.
+    11. All that's left to do now is enable this lambda function to trigger on S3 object creation. Click **Add trigger** to start setting that up.
+    12. From the dropdown, select **S3** as your source.
+    13. Enter the name of the S3 bucket you'd like to apply the metadata to in the **Bucket** field.
+    14. Remove the default **All object create events** from the event types. From the Event types dropdown, select **PUT**.
+    15. Check the **Recursive invocation** box, then click **Add** in the bottom right.
+
+    The Lambda function will now start automatically adding the compression metadata to all newly-added S3 objects.
+  </Collapser>
+</CollapserGroup>
+
+### Automatic Decompression in Azure
+
+If you're exporting data to Azure, it's possible to view decompressed versions of the objects stored in your event hub using a [Stream Analytics Job](https://learn.microsoft.com/en-us/azure/event-hubs/process-data-azure-stream-analytics). To do so, follow these steps:
+
+1. Follow [this guide](https://learn.microsoft.com/en-us/azure/event-hubs/process-data-azure-stream-analytics) up to step 16.
+ * On step 13, you may choose to use the same event hub as the output without breaking anything, though we do not recommend this if you intend to proceed to step 17 and start the job, as doing so as not been tested.
+2. Navigate to "inputs", and click on the input you set up.
```

Comment:
@ecchilds Is this in a UI? If so, it should be bolded, and include a preface (i.e., "In AWS, navigate to **Inputs**...")

Reply:
Alright, I'll put together another commit to make that change.

Reply:
Done.